docs/api.rst

```diff
@@ -2608,6 +2608,50 @@ of :class:`enum.Enum`.
 
         The guild may contain NSFW content.
 
+.. class:: EntityType
```

Comment:
This should probably start with ScheduledEvent or does it not do that in the docs?